### DISCUSSION TITLE
If you could, could you add a tab scroller?

### DISCUSSION THREAD

**Discussion by Snowblind019** [zen-browser/desktop#3000] [Ideas]
Floorp, another firefox fork, has a option where you can use your scroll wheel to scroll through different tabs. I am wondering if it's possible to add such a feature to the vertical tabs of Zen?

**Comment by pinage404** [comment]
I think what you are looking for already exists https://docs.zen-browser.app/faq#how-to-switch-tabs-by-scrolling ?

**Reply by Snowblind019** [reply]
Oh thank you, I didn't know about that. Much obliged!

**Comment by pinage404** [comment]
Related https://github.com/zen-browser/desktop/discussions/1429 https://github.com/zen-browser/desktop/discussions/1897 https://github.com/zen-browser/desktop/issues/1541
